packages/js/uri-resolver-extensions/src/ExtendableUriResolver.ts

```diff
@@ -18,11 +17,20 @@ export class ExtendableUriResolver extends UriResolverAggregatorBase<
   Error,
   Error
 > {
-  private readonly resolverName: string;
+  public static interfaceUri: Uri = new Uri(
```

Comment:
`extensionInterfaceUri` maybe?
`interfaceUri` kind of implies it's the interface of the ExtendableUriResolver instead of the resolve extension

Reply:
Agreed, this does look much nicer.